### ISSUE TITLE
overwriting config parameters from the command line

### ISSUE THREAD

**Issue by curly1** [mravanelli/pytorch-kaldi#47]
Hi,
I was trying to train a new model with a different seed by overwriting config parameters from the command line with this command:
`python run_exp.py cfg/my_cfg --exp,out_folder="exp/new_dir" --exp,seed=1234`
As a result, `exp_new_dir` was created with `exp_files` copied from the original model, a new model wasn't trained, and the information for the original model from `cfg/my_cfg` was printed out. Can the `exp,out_folder` be specified this way to train and store a new model with modified options?
Thanks,
Joanna

**Comment by mravanelli** [comment]
Hi Joanna,
thank you very much for let us know this issue. I tried it and seems there
is something weird happening. We will try to fix it as soon as possible.
Meanwhile I would suggest you to modify the cfg_file directly.



Mirco

**Comment by mravanelli** [comment]
We will keep you updated!

Best,

Mirco

**Comment by TParcollet** [comment]
We have found the problem ! Thanks you very much for pointing it out. It will be patched on the next commit.

**Comment by mravanelli** [comment]
Hi Joanna, 
if this could be interesting for you, we have just updated our toolkit. Now the toolkit allows parallel data loading and should be significantly faster. Moreover, one can now use the token "N_out_lab_mono" and the toolkit automatically detects the number of monophone targets (In other words, you don't have to do it manually anymore). We also corrected some minor bugs, including the command line error you pointed us. Let me thank you for the feedback! Of course, if you have the chance to take a look and try the latest version it would be great to have again your feedback/comments.
